### PR TITLE
Xeno Health Alert Improvements

### PR DESCRIPTION
**Pull request by Surrealaser**
## About The Pull Request

1. Xeno hive health alert % decreased from 33% to 25%
2. Preference settings made more intuitive; previously the wording was misleading.
3. Xeno hive alerts can now visibly ping non-turf targets
4. Various minor other improvements

## Why It's Good For The Game

Improves the functionality of Xeno Health alerts and Xeno Hive alerts in general.

## Changelog
:cl:
code: Xeno hive alert code improved; can now visibly ping non-turf targets.
tweak: Xeno health alert preferences made more intuitive.
/:cl:

### CODE REVIEW

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/temporary_visuals/temporary_visual.dm [hunk 1]
```diff
@@ -34,3 +34,43 @@
...
+	///The visual effect we're attaching
+	var/image/holder
+
+/obj/effect/temp_visual/xenomorph/xeno_tracker_target/Destroy()
+	. = ..()
+	if(tracker_target && holder) //Check to avoid runtimes
+		tracker_target.overlays -= holder //remove the overlay
+	tracker_target = null //null the target var
+	QDEL_NULL(holder) //remove the holder
+	holder = null //null the holder var
```
[the remark this answers is not in the document]
Comment (Surrealaser) [reply code/game/objects/effects/temporary_visuals/temporary_visual.dm:58]: Qdel deletes the holder, but it does not null the holder var saved to the obj

**Comment by BraveMole** [comment]
Can you make it so ghosts have in the hive rally alert the icon of the caster, and the not the icon of the target? Having turfs icons show up is meh

**Comment by Surrealaser** [comment]
> Can you make it so ghosts have in the hive rally alert the icon of the caster, and the not the icon of the target? Having turfs icons show up is meh

This doesn't actually alert ghosts, and if I were to change the ghost ping for hive alerts I'd probably get yelled at for not atomizing

File: code/game/objects/effects/temporary_visuals/temporary_visual.dm (hunk 1 above)
Comment (SplinterGP) [inline code/game/objects/effects/temporary_visuals/temporary_visual.dm:58]:
```suggestion
	QDEL_NULL(holder) //remove the holder and null the var
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/temporary_visuals/temporary_visual.dm (hunk 1 above)
Comment (Surrealaser) [reply code/game/objects/effects/temporary_visuals/temporary_visual.dm:58]: I'm fairly certain this doesn't work, but I'll do it to get the merge

**Comment by Surrealaser** [comment]
@SplinterGP Needs re-TM

**Comment by TiviPlus** [comment]
Rogue debug, review stuff before TM @SplinterGP + Have you fixed the issues with player preferences being reset/ignored

**Comment by Surrealaser** [comment]
> Rogue debug, review stuff before TM @SplinterGP + Have you fixed the issues with player preferences being reset/ignored

Yes, should be addressed.

**Review by TiviPlus — APPROVED** [review]

**Comment by TiviPlus** [comment]
Fix the issues with alert prefs not working with previous users

**Comment by Surrealaser** [comment]
> Fix the issues with alert prefs not working with previous users

It defaults to off.

Since people apparently don't want to or know how to toggle off alerts, how do I change their preferences to default to off?

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**